### PR TITLE
Add git reflog exists subcommand

### PR DESCRIPTION
**Pull request by driusan**
This is the simplest subcommand of the reflog command.

### CODE REVIEW

File: cmd/reflog.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		return fmt.Errorf("reflog subcommand %v not implemented", subcmd)
+	case "exists":
+		if len(args) != 2 {
+			return fmt.Errorf("usage: %v reflog exists <ref>", os.Args[0])
+		}
+		if git.ReflogExists(c, git.Refname(args[1])) {
+			os.Exit(0)
+		}
+		os.Exit(1)
+		// without this git complains about no return at the end of
```
Comment (sirnewton01) [inline cmd/reflog.go]: s/git/go/ ?

File: git/reflog.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package git
+
+// Returns true if a reflog exists for refname r under client.
+func ReflogExists(c *Client, r Refname) bool {
+	return c.GitDir.File(File(r)).Exists()
```
Comment (sirnewton01) [inline git/reflog.go]: I thought that the ref logs are the files that are typically in .git/logs/refs/... that store a log of the changes to the ref. Shouldn't this check if c.GitDir.File(File("logs")).File(File(r)).Exists()?

Comment (driusan) [reply git/reflog.go]: oops.. you're right.

File: cmd/reflog.go (hunk 1 above)
Comment (driusan) [reply cmd/reflog.go]: fixed

File: git/reflog.go (hunk 2 above)
Comment (driusan) [reply git/reflog.go]: should be fixed now

**Pull request merged**